web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1634,9 +1644,39 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
   @volatile var cometGetTimeout = 140000
 
   /**
-   * Compute the headers to be sent to the browser in addition to anything else that's sent
-   */
-  val supplementalHeaders: FactoryMaker[List[(String, String)]] = new FactoryMaker(() => List(("X-Lift-Version", liftVersion), ("X-Frame-Options", "SAMEORIGIN"))) {}
+   * Compute the headers to be sent to the browser in addition to anything else
+   * that's sent.
+   *
+   * Note that the headers for the applications `SecurityRules` are also set
+   * here, so if you override the supplemental headers, you should
+   * either refer back to the default set or make sure to include
+   * `LiftRules.securityRules.headers`.
+   */
+  val supplementalHeaders: FactoryMaker[List[(String, String)]] = new FactoryMaker(() => {
+    ("X-Lift-Version", liftVersion) ::
+    ("X-Frame-Options", "SAMEORIGIN") ::
```

Comment:
To clarify, Content-Security-Policy 2 does feature a `frame-ancestors` flag for this… But I'm aiming at CSP 1 support here for the time being.